README.md

```diff
@@ -59,87 +51,90 @@ After this, you have some options:
+other options, feel free to explore and please do reach out if you learn anything 
+interesting or would like to make a contribution.
 
-*Windows*
-```
-> set GUROBI_JL_SKIP_LIB_CHECK=1
-```
+### Julia Environment
 
-Or, if you don't have Xpress installed on your system, set the XPRESS_JL_SKIP_LIB_CHECK environment variable:
+Running HEM requires access to LP and MIP solvers. Because different choices can 
+be made in this regard, we recommend setting up your computational environment 
+based on one of the environments under [runner](https://github.com/nrel-hem/HolisticElectricityModel.jl/tree/main/runner).
 
-*Linux/Mac*
-```bash
-$ export XPRESS_JL_SKIP_LIB_CHECK=1
-```
+To set up such an environment from the Julia terminal/REPL (REPL stands for 
+read-eval-print loop):
 
-*Windows*
-```
-> set XPRESS_JL_SKIP_LIB_CHECK=1
+```julia
+julia> ]
+(@v1.10) pkg> activate runner/Xpress # select desired solver at this point
+(Xpress) pkg> dev --local . ../HolisticElectricityModelData.jl
+(Xpress) pkg> instantiate
+# Hit Backspace
+julia> 
 ```
 
-Second, install HolisticElectricityModel.jl and HolisticElectricityModelData.jl in the runner project:
+Using Gurobi is the same as using Xpress, except that there is an extra 
+`build Gurobi` step:
 
-```bash
-> julia
-```
 ```julia
 julia> ]
-pkg> activate runner
-pkg> dev . ../HolisticElectricityModelData.jl
+(@v1.10) pkg> activate runner/Gurobi # select desired solver at this point
+(Gurobi) pkg> dev --local . ../HolisticElectricityModelData.jl
+(Gurobi) pkg> instantiate
+(Gurobi) pkg> build Gurobi
+# Hit Backspace
+julia> 
 ```
 
-#### REPL
-
-Open the REPL from the HolisticElectrictyModel.jl directory. Then:
+Alternatively, the Julia environment can be specified on the command line, either
+to open the REPL:
 
 ```julia
+> julia --project=runner/Xpress
+# If you want to check that your environment is in working order before proceeding
 julia> ]
-pkg> activate runner
+(Xpress) > instantiate
 # Hit Backspace
-julia> include("script/driver_xpress.jl") # or include("script/driver_gurobi.jl")
+julia> 
 ```
 
-or, if you specify the runner project when you open the REPL:
-
+or to directly run a script:
 ```bash
 > cd ~/HolisticElectricityModel.jl
-> julia --project=runner
+> julia --project=runner/Gurobi script/hem.jl
 ```
 
-and if the runner environment is already set up then you can simply:
+### Performing a Model Run
 
-```julia
-julia> include("script/driver_xpress.jl") # or include("script/driver_gurobi.jl")
-```
-
-#### Command Line
 
-```bash
-> cd ~/HolisticElectricityModel.jl
-> julia --project=runner script/driver_xpress.jl # or script/driver_gurobi.jl
-```
 
-### Tests
+### Running Tests
 
 Specify a scenario you want to test by setting `driver_name` as desired at the top of `test/test_driver.jl` and then editing the driver file accordingly. Currently the model outputs available to test against are in the `test/driver_outputs/ba_1_base_2018_future_2_ipps_1` directory created with input parameters:
```

Comment:
This should also change based on using config files and `hem.jl`

Reply:
Updated the readme. I think after merging in main some of the local changes got overwritten.